### PR TITLE
updated Abstract and Introduction

### PR DESCRIPTION
**Pull request by nitedog**
fleshed out the Abstract and Introduction for first publication.

### CODE REVIEW

**Comment by nitedog** [comment]
That's fine for now. Just remembered it may be "ACT-framework" instead. But don't worry about that for now.

**Comment by moekraft** [comment]
Hi Shadi, I did a review yesterday, however, my comments were still pending. I just hit submit review and looks like they are no longer pending. Sorry about that. Moe P.S. Let me know if you would like me to make these updates.

**Comment by moekraft** [comment]
I have incorporated my review changes into Shadi's updates to the Abstract and Introduction. Here is a preview of the HTML for review https://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/w3c/wcag-act/b89b3e07c0e88529a3843c917e545999d67a8a52/NOTE-act-rules-common-aspects.bs

**Comment by moekraft** [comment]
This PR addresses Issue #318 It's ready for review.

File: NOTE-act-rules-common-aspects.bs [hunk 1]
```diff
@@ -9,16 +9,20 @@ TR: https://www.w3.org/TR/
...
+The term [Aspects Under Test](https://w3c.github.io/wcag-act/act-rules-format.html#input-aspects) is defined by the [Accessibility Conformance Testing (ACT) Rules Format 1.0 Specification](https://w3c.github.io/wcag-act/act-framework.html). An aspect is a distinct part of a [test subject](https://w3c.github.io/wcag-act/act-rules-format.html#test-subject). Atomic rules are required to list aspects under test in the [applicability](https://w3c.github.io/wcag-act/act-rules-format.html#applicability) and [expectations](https://w3c.github.io/wcag-act/act-rules-format.html#expectations). 
+
+Some aspects are already well defined in a formal specification within the context of web content, such as HTTP messages, DOM tree, and CSS styling [css-2018]. These do not warrant a detailed description in ACT Rules Format 1.0 specification. Instead, these are listed in this informative document, which can be updated more easily. Atomic rules can refer to one of these common aspects under test, however, these common aspects are not required to conform to the ACT Rules Format 1.0 specification.
+
+Examples of ACT Rules can be in the [ACT Rules Repository](https://w3c.github.io/wcag-act-rules/).
```
Comment (annethyme) [inline NOTE-act-rules-common-aspects.bs]: "can be" --> "can be found"

File: NOTE-act-rules-common-aspects.bs [hunk 2]
```diff
@@ -9,16 +9,20 @@ TR: https://www.w3.org/TR/
...
 Markup Shorthands: markdown yes
 </pre>
 
 Introduction {#intro}
 =====================
 
-<p class="note" role="note">
-"Editor's note: **DRAFT** This note captures Common Aspects Under Test. The TR metadata needs to be updated when ACT rules has a TR url."
-</p>
+The term [Aspects Under Test](https://w3c.github.io/wcag-act/act-rules-format.html#input-aspects) is defined by the [Accessibility Conformance Testing (ACT) Rules Format 1.0 Specification](https://w3c.github.io/wcag-act/act-framework.html). An aspect is a distinct part of a [test subject](https://w3c.github.io/wcag-act/act-rules-format.html#test-subject). Atomic rules are required to list aspects under test in the [applicability](https://w3c.github.io/wcag-act/act-rules-format.html#applicability) and [expectations](https://w3c.github.io/wcag-act/act-rules-format.html#expectations).
```
Comment (maryjom) [inline NOTE-act-rules-common-aspects.bs]: Again, "Aspects Under Test" should be "Input Aspects".

File: NOTE-act-rules-common-aspects.bs [hunk 3]
```diff
@@ -9,16 +9,20 @@ TR: https://www.w3.org/TR/
...
+
+Some aspects are already well defined in a formal specification within the context of web content, such as HTTP messages, DOM tree, and CSS styling [css-2018]. These do not warrant a detailed description in ACT Rules Format 1.0 specification. Instead, these are listed in this informative document, which can be updated more easily. Atomic rules can refer to one of these common aspects under test, however, these common aspects are not required to conform to the ACT Rules Format 1.0 specification.
+
+Examples of ACT Rules can be in the [ACT Rules Repository](https://w3c.github.io/wcag-act-rules/).
+
+The aspects listed in this document can be used by authors of ACT Rules, to refer to common types more easily. This improves the development process and supports consistency across rules. This list could be extended and refined at any time, for example to include popular aspects or to provide clarification. Existing aspects could also be marked as obsoleted, if needed.
```
Comment (tbostic32) [inline NOTE-act-rules-common-aspects.bs]: 1. Remove comma: "ACT Rules, to refer" --> "ACT Rules to refer"
2. "Could" to "can" to keep tense consistency: "This list could " --> "This list can"; "Existing aspects could " --> "Existing aspects can "

File: NOTE-act-rules-common-aspects.bs [hunk 4]
```diff
@@ -9,16 +9,20 @@ TR: https://www.w3.org/TR/
...
-</p>
+The term [Aspects Under Test](https://w3c.github.io/wcag-act/act-rules-format.html#input-aspects) is defined by the [Accessibility Conformance Testing (ACT) Rules Format 1.0 Specification](https://w3c.github.io/wcag-act/act-framework.html). An aspect is a distinct part of a [test subject](https://w3c.github.io/wcag-act/act-rules-format.html#test-subject). Atomic rules are required to list aspects under test in the [applicability](https://w3c.github.io/wcag-act/act-rules-format.html#applicability) and [expectations](https://w3c.github.io/wcag-act/act-rules-format.html#expectations). 
+
+Some aspects are already well defined in a formal specification within the context of web content, such as HTTP messages, DOM tree, and CSS styling [css-2018]. These do not warrant a detailed description in ACT Rules Format 1.0 specification. Instead, these are listed in this informative document, which can be updated more easily. Atomic rules can refer to one of these common aspects under test, however, these common aspects are not required to conform to the ACT Rules Format 1.0 specification.
```
[the remark this answers is not in the document]
Comment (moekraft) [reply NOTE-act-rules-common-aspects.bs]: I think it's okay to title Common Input Aspects.

File: NOTE-act-rules-common-aspects.bs [hunk 5]
```diff
@@ -9,16 +9,20 @@ TR: https://www.w3.org/TR/
 Group: act-framework
 Editor: Wilco Fiers, Deque Systems
 Editor: Maureen Kraft, IBM Corp.
-Abstract: Working Group note to identify common aspects under test.
+Abstract: This document is a companion to the [Accessibility Conformance Testing (ACT) Rules Format 1.0](https://w3c.github.io/wcag-act/act-framework.html) specification. It lists Common Aspects Under Test as defined by the ACT Rules Format 1.0 specification. This document is informative. It provides a reference to well defined aspects to assist authors in writing ACT Rules and to support the consistency of ACT Rules.
```
[the remark this answers is not in the document]
Comment (moekraft) [reply NOTE-act-rules-common-aspects.bs]: Updated specification to replace aspects under test with input aspects. Please review, https://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/w3c/wcag-act/b7ea4f761206c8a5771f1a5b3ade8d18f5e47f85/NOTE-act-rules-common-aspects.bs

File: NOTE-act-rules-common-aspects.bs (hunk 2 above)
Comment (moekraft) [reply NOTE-act-rules-common-aspects.bs]: https://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/w3c/wcag-act/b7ea4f761206c8a5771f1a5b3ade8d18f5e47f85/NOTE-act-rules-common-aspects.bs

**Comment by moekraft** [comment]
@tbostic32 @maryjom @annethyme @nitedog Thanks for the feedback. I made the suggested updates. Please review https://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/w3c/wcag-act/b7ea4f761206c8a5771f1a5b3ade8d18f5e47f85/NOTE-act-rules-common-aspects.bs

**Comment by moekraft** [comment]
@WilcoFiers I have incorporated the feedback into the Common Input Aspects WG note. Latest can be reviewed at. https://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/w3c/wcag-act/8cfb95bfb3ea8d4c40cd25b4e66a47d1712f32c8/NOTE-act-rules-common-aspects.bs Let me know if this is okay to merge.

File: NOTE-act-rules-common-aspects.bs [hunk 6]
```diff
@@ -9,47 +9,56 @@ TR: https://www.w3.org/TR/
...
+Some input aspects are already well defined in a formal specification within the context of web content, such as HTTP messages, DOM tree, and CSS styling [[css-2018]]. These do not warrant a detailed description in ACT Rules Format 1.0 specification. Instead, these are listed in this informative document, which can be updated more easily. Atomic rules can refer to one of these common input aspects, however, these common input aspects are not required to conform to the ACT Rules Format 1.0 specification.
 
+Examples of ACT Rules can be found in the [ACT Rules Repository](https://w3c.github.io/wcag-act-rules/).
+
+The input aspects listed in this document can be used by authors of ACT Rules to refer to common types more easily. This improves the development process and supports consistency across rules. This list can be extended and refined at any time, for example, to include popular input aspects or to provide clarification. Existing input aspects can also be marked as obsoleted, if needed.
+
+
+
+Common Input Aspects {#input-aspects-common}
+============================================
```
Comment (nitedog) [inline NOTE-act-rules-common-aspects.bs]: (minor) I don't see any benefit in this additional layer in the headings structure but I don't feel strongly one way or the other. Also happy to log this as an issue, and come back to it after publication.

**Comment by nitedog** [comment]
Looks good to me, many thanks for all the work on this @moekraft!

**Comment by moekraft** [comment]
@WilcoFiers @nitedog @maryjom Should I merge this pull request or wait until the Roll Back #328 is complete?

**Pull request merged**